src/components/TopBar/ActivityIndicator.js

```diff
@@ -123,6 +126,23 @@ class ActivityIndicatorInner extends PureComponent<Props, State> {
         </Box>
       </ItemContainer>
     )
+
+    if (error) {
+      return (
+        <Tooltip
+          tooltipBg="alertRed"
+          render={() => (
+            <Box fontSize={4} p={2} style={{ maxWidth: 250 }}>
+              <TranslatedError error={error} />
+            </Box>
```

Comment:
could use a variable as well because the inner don't change

Reply:
actually maybe you don't need a render function , just a dom , like children ?

Reply:
it's ok for now, i'll merge

Reply:
+1 we could refactor all tooltips passing nodes directly